suitcase/hdf5.py

```diff
@@ -2,21 +2,32 @@
 import warnings
 import h5py
 import json
+import copy
 from metadatastore.commands import get_events_generator
 from databroker.databroker import fill_event
 from databroker.core import Header
 
 
 __version__ = "0.2.2"
 
-def export(headers, filename):
+def export(headers, filename, stream_name=None, fields_unwanted=None, timestamps_opt=True):
     """
+    Create hdf5 file to preserve the structure of databroker. Necessary copy are made at each step, so
+    headers will not be modified.
+
     Parameters
     ----------
     headers : a Header or a list of Headers
         objects retruned by the Data Broker
     filename : string
         path to a new or existing HDF5 file
+    stream_name : string, optional
+        None means save all the data from each descriptor, i.e., user can define stream_name as primary,
+        so only data with descriptor.name == primary will be saved.
+    fields_unwanted : list, optional
+        list of names which are excluded when data is transfered to HDF5 file
+    timestamps_opt : Bool, optional
```

Comment:
The bundling time of the event is literally when the event document gets created on the client side before it is inserted into metadatastore.

[here] (https://github.com/NSLS-II/bluesky/blob/984fc0e4bfb0e3db2d6bf307e7cfa7eb5677c1c6/bluesky/run_engine.py#L1201) is the code in bluesky that emits the event document which gets consumed by the metadatastore insertion callback which is registered [here] (https://github.com/NSLS-II/bluesky/blob/5b50641e40914e34c592c4b6eb3f0c52f2423fc9/bluesky/register_mds.py#L64).  Note that the time is created right before the event document is emitted: https://github.com/NSLS-II/bluesky/blob/984fc0e4bfb0e3db2d6bf307e7cfa7eb5677c1c6/bluesky/run_engine.py#L1198